CMakeLists.txt

```diff
@@ -629,6 +629,38 @@ if (APPLE)
 	target_link_libraries(torrent-rasterbar PRIVATE "-framework CoreFoundation" "-framework SystemConfiguration")
 endif()
 
+# check if we need to link with libatomic (not needed on MSVC)
+if (NOT MSVC)
+	# TODO: migrate to CheckSourceCompiles in CMake >= 3.19
+	include(CheckCXXSourceCompiles)
+
+	set(ATOMICS_TEST_SOURCE [=[
+		#include <atomic>
+		#include <cstdint>
+		std::atomic<int> x{0};
+		int main() {
+			x.fetch_add(1, std::memory_order_relaxed);
+			return 0;
+		}
+	]=])
+	string(REPLACE "std::atomic<int>" "std::atomic<int64_t>" ATOMICS64_TEST_SOURCE "${ATOMICS_TEST_SOURCE}")
```

Comment:
technically, `int64_t` needs `std::` in front of it. If the header `<stdint.h>` had been included, it wouldn't be needed, but this program includes the C++ version of that header.

Reply:
fixed.